modules/web-console/frontend/app/configuration/generator/generator/JavaTransformer.service.js

```diff
@@ -514,11 +520,12 @@ export default class IgniteJavaTransformer extends AbstractTransformer {
 
     static _constructMap(sb, map, vars = []) {
         const keyClsName = this.javaTypes.shortClassName(map.keyClsName);
+        const keyClsGeneric = map.keyClsGenericName ? `<${map.keyClsGenericExtends ? '? extends ' : ''}${this.javaTypes.shortClassName(map.keyClsGenericName)}>` : '';
```

Comment:
Please try to simplify this expression, may be will be better to separate for two lines or add braces.

Reply:
Expression simplified.